zoo-css-challenge/style.css

```
@@ -187,9 +223,12 @@ footer {
 .membership strong {
   color: #ff7f11;
 }
-
+.membership p {
+  max-width:700px;
+  margin: 30px 280px 30px 280px;
```

```suggestion
  margin: auto;
```
Alternatively, we can use `auto` to centre the element. `auto` distributes the space around the element evenly